internal/bridge/crunchybridgecluster/crunchybridgecluster_controller.go

```diff
@@ -466,24 +518,23 @@ func (r *CrunchyBridgeClusterReconciler) reconcileBridgeConnectionSecret(
 	if err != nil {
 		meta.SetStatusCondition(&crunchybridgecluster.Status.Conditions, metav1.Condition{
 			ObservedGeneration: crunchybridgecluster.GetGeneration(),
-			Type:               v1beta1.ConditionCreating,
-			Status:             metav1.ConditionFalse,
+			Type:               v1beta1.ConditionReady,
+			Status:             metav1.ConditionUnknown,
 			Reason:             "SecretInvalid",
 			Message: fmt.Sprintf(
-				"Cannot create with bad secret: %v", "TODO(crunchybridgecluster)"),
+				"Cannot create with bad secret: %v", err),
+		})
+		meta.SetStatusCondition(&crunchybridgecluster.Status.Conditions, metav1.Condition{
+			ObservedGeneration: crunchybridgecluster.GetGeneration(),
+			Type:               v1beta1.ConditionUpgrading,
+			Status:             metav1.ConditionUnknown,
+			Reason:             "UpgradeConditionUnknown",
+			Message:            "The condition of the upgrade(s) is unknown.",
```

Comment:
Maybe work into the message the fact that the condition of the upgrade is unknown because of the bad secret.